x-pack/plugins/ml/server/saved_objects/initialization/space_overrides.test.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IScopedClusterClient } from 'kibana/server';
+import { createJobSpaceOverrides } from './space_overrides';
+
+const jobs = [
+  {
+    job_id: 'kibana-logs-ui-default-default-log-entry-rate',
+  },
+  {
+    job_id: 'kibana-logs-ui-other_space-default-log-entry-rate',
+  },
+  {
+    job_id: 'kibana-logs-ui-other_space-default-log-entry-categories-count',
+  },
+  {
+    job_id: 'kibana-logs-ui-other_space-internal-stack-monitoring-log-entry-rate',
+  },
+  {
+    job_id: 'kibana-logs-ui-other_space-dinosaur-log-entry-rate', // shouldn't match
+  },
+  {
+    job_id: 'kibana-logs-ui-other_space-default-dinosaur', // shouldn't match
+  },
+];
+
+const result = {
+  overrides: {
+    'anomaly-detector': {
+      'kibana-logs-ui-default-default-log-entry-rate': ['default'],
+      'kibana-logs-ui-other_space-default-log-entry-rate': ['other_space'],
+      'kibana-logs-ui-other_space-default-log-entry-categories-count': ['other_space'],
+      'kibana-logs-ui-other_space-internal-stack-monitoring-log-entry-rate': ['other_space'],
+    },
+    'data-frame-analytics': {},
+  },
+};
+
+const callAs = {
+  ml: {
+    getJobs: jest.fn(() =>
+      Promise.resolve({
+        body: { jobs },
+      })
+    ),
+  },
+};
+
+const mlClusterClient = ({
+  asInternalUser: callAs,
+} as unknown) as IScopedClusterClient;
+
+describe('ML - data recognizer', () => {
```

Comment:
Are these descriptions copy and pasted from another file? Shouldn't they refer to space overrides?

Reply:
Updated in https://github.com/elastic/kibana/pull/83127/commits/44f31c262f41dbfa7d579657e1310e96e60b5814